### ISSUE TITLE
Add skeleton states for different card components

### ISSUE THREAD

**Issue by sebastinez** [radicle-dev/radicle-interface#376]
To reduce the jarringness (what a word 😵) of the display and give the components a loading substitute I propose to replace `Loading` component in some specific places like the Landing page or the project listing of a seed node with pulsating skeleton cards, which are then replaced by the final loaded component.

Here an example video

https://user-images.githubusercontent.com/7912302/189319489-9244e86c-e0d6-4e6e-8584-b50df8c5d4f1.mov

**Comment by efstajas** [comment]
just one quick thought on the example vid, ideally the skeleton would crossfade with the content for something like `.3s` once the content is loaded :) 

Maybe this is useful — I've found that a good way to achieve this is to have a wrapper component with a `loading` prop, and all the card content passed in as a slot. If `loading`, that wrapper just shows the pulsating placeholder, once `loading === false` it fades to the real content. I've found this is the most convenient structure for achieving this, and easily allows wrapping all kinds of different content into a skeleton placeholder representation. Also, it avoids having a global "the view is loading" state variable — instead allowing views comprised of multiple parts that make their own independent resource requests and fade to content independently.
